src/api/app/policies/user_policy.rb

```diff
@@ -10,4 +10,8 @@ def show?
   def check_watchlist?
     record.login == User.session!.login || User.admin_session?
   end
+
+  def comments?
```

Comment:
@hellcp-work just curious, why shouldn't we display the comments to the user that owns them and is in the beta program?

Reply:
and a suggestion, just `comments?` is a little confusing for me. By the name I dont really get what it is suppose to do, maybe `list_comments?`